x-pack/plugins/observability/public/pages/slo_details/slo_details.test.tsx

```diff
@@ -8,6 +8,158 @@
 import React from 'react';
 import { screen } from '@testing-library/react';
 
+import { useKibana } from '../../utils/kibana_react';
```

Comment:
These changes seemed to be duplicated by accident, would you please revert them?